### PR TITLE
Create the Kdump DBus service

### PR DESCRIPTION
**Pull request by poncovka**
The current support for Anaconda add-ons will be removed.
Use the support for Anaconda DBus add-ons instead.

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Comment by poncovka** [comment]
The related fixes were released in `anaconda-34.13-1`.

**Comment by poncovka** [comment]
@ryncsn, can I ask you for a review, please?

File: com_redhat_kdump/gui/spokes/kdump.py [hunk 1]
```diff
@@ -50,12 +51,12 @@ class KdumpSpoke(NormalSpoke):
 
     @classmethod
     def should_run(cls, environment, data):
-        # the KdumpSpoke should run only if requested
-        return kernel_arguments.is_enabled("kdump_addon")
+        return is_module_available(KDUMP)
```
Comment (ryncsn) [inline com_redhat_kdump/gui/spokes/kdump.py:54]: Hi, previously this addon only work if kdump_addon is specified in kernel cmdline, will this change the behavior?

Comment (ryncsn) [reply com_redhat_kdump/gui/spokes/kdump.py:54]: Ah, never mind, I saw the kernel_arguments.is_enabled("kdump_addon"): in the service py file below.

**Comment by poncovka** [comment]
Thank you! It should be hopefully all for a while. Could you release a new version for rawhide, please?

**Comment by ryncsn** [comment]
Sure, I've just made a new release in Rawhide.

**Pull request merged**